homeassistant/components/kaleidescape/manifest.json

```diff
@@ -9,7 +9,7 @@
     }
   ],
   "documentation": "https://www.home-assistant.io/integrations/kaleidescape",
-  "requirements": ["pykaleidescape==2022.2.6"],
+  "requirements": ["pykaleidescape==1.0.0"],
```

Comment:
Needed to update the pypi library for this PR, so changing the versioning scheme while I can (date based really makes no sense).